api-lib/sendSocialMessage.ts

```diff
@@ -0,0 +1,69 @@
+import fetch from 'node-fetch';
+
+import { TELEGRAM_BOT_BASE_URL } from './config';
+import { DISCORD_BOT_NAME, DISCORD_BOT_AVATAR_URL } from './constants';
+import { gql } from './Gql';
+
+type SocialMessage = {
+  message: string;
+  circleId: number;
+  sanitize: boolean;
+  channels: {
+    discord?: boolean;
+    telegram?: boolean;
+  };
+};
+
+function cleanStr(str: string) {
+  return str.replace(/:|-|\/|\*|_|`/g, '');
+}
+
+export async function sendSocialMessage({
+  message,
+  circleId,
+  sanitize = true,
+  channels,
+}: SocialMessage) {
+  const msg = sanitize ? cleanStr(message) : message;
+
+  const {
+    circles_by_pk: { discord_webhook, telegram_id },
+  } = await gql.getCircle(circleId);
```

Comment:
@exrhizo I now used the admin client here. Do you want me to use it in `check-nominee.ts` for the query and mutation as well?

Reply:
yeap , @jsur please do so

Reply:
Cool, will take a look today.

Reply:
Added @zashton

Reply:
Added also to `handleCheckNomineeMsg`